solr/contrib/ltr/src/test/org/apache/solr/ltr/ranking/TestModelQuery.java

```diff
@@ -141,28 +141,28 @@ public void testModelQueryEquality() throws ModelException {
         "testModelName",
         features, norms, "testStoreName", allFeatures, modelParams);
 
-    final ModelQuery m1 = new ModelQuery(algorithm1);
-    final ModelQuery m2 = new ModelQuery(algorithm1);
-
-    assertEquals(m1, m2);
-    assertEquals(m1.hashCode(), m2.hashCode());
+    final ModelQuery m0 = new ModelQuery(algorithm1);
```

Comment:
No, the old code checked the same 2 objects and their differences when calling setEfi.  Since now it is part of the constructor, I had to make a 3rd object without efi to do comparisons against.